lib/order.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'menu'
+
+module Order
+
+include Menu
+
+  def total_price
+    menu_prices
+    @overall_price = @unit_prices.inject(0) {|sum, i|  sum + i }
+  end
+
+  def menu_prices
+    @unit_prices = []
```

Comment:
you do not need `@unit_prices` to be an instance variable.  Avoid unnecessary state in your objects - just return the value from the method and let the calling code decide what to do with it.